pkg/components/microservice.go

```diff
@@ -54,12 +55,14 @@ func newMicroservice(
 	image string,
 	instanceCount int32,
 	generators map[string]ytconfig.GeneratorDescriptor,
-	deploymentName, serviceName string) microservice {
+	deploymentName, serviceName string,
+	tolerations []corev1.Toleration) microservice {
 	return &microserviceImpl{
 		labeller:      labeller,
 		image:         image,
 		instanceCount: instanceCount,
 		ytsaurus:      ytsaurus,
+		tolerations:   tolerations,
 		service: resources.NewHTTPService(
 			serviceName,
 			nil,
```

Comment:
but need to pass tolerations in NewDeployment instead.

Reply:
fixed